crates/nu-cli/src/deserializer.rs

```diff
@@ -92,7 +92,7 @@ impl<'de, 'a> de::Deserializer<'de> for &'a mut ConfigDeserializer<'de> {
     {
         unimplemented!("deserialize_any")
     }
-    fn deserialize_bool<V>(self, visitor: V) -> Result<V::Value, Self::Error>
+    fn deserialize_bool<V>(self, _visitor: V) -> Result<V::Value, Self::Error>
```

Comment:
Same comment with these. Seems like they're used. We're you setting warnings?

Reply:
Oh ya. Good catch. I forgot I took a trip through deserialize land. These should be fixed now.